tower.yml

```diff
@@ -0,0 +1,33 @@
+reports:
```

Comment:
Was not aware of this tower feature, guess that reports are rendered online, is it available in the free version?

Reply:
🤫 It should be available on `tower.nf` yep. We have been playing with it for a bit and it is relatively stable so thought I would push it here to test it out from a real pipeline!